### ISSUE TITLE
Saving and loading settings in yaml files

### ISSUE THREAD

**Issue by gitfvb** [Apteco/AptecoPSFramework#4]
This looks promising: https://github.com/cloudbase/powershell-yaml

```PowerShell
$settings | ConvertTo-Yaml | Set-Content -path "settings.yaml" -Encoding UTF8 
$s = get-content -Path settings.yaml -Encoding UTF8 -Raw | ConvertFrom-Yaml
```

But the second command gives us a hashtable back instead of a pscustomobject, which could also be handled, but needs a second thought.

Good hints on the second topic: https://stackoverflow.com/questions/72218329/unable-to-convert-yaml-to-powershell-object

So this should work

```PowerShell
[pscustomobject]$os_list = (ConvertFrom-Yaml -yaml (get-content -Raw C:\Powershell\TestCSVs\variables.stg.yml))
[pscustomobject]$os_list = ConvertTo-Yaml $os_list -JsonCompatible
````

or this, too

```PowerShell
$documentWithVariables = Get-Content -Path ..\variables.stg.yml -Raw |ConvertFrom-Yaml 
$oData = [PSCustomObject]$documentWithVariables.variables
```

More notes about using comments
https://github.com/aaubry/YamlDotNet/issues/534

Use this in vscode
https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml

Link these dependencies directly
Package: https://github.com/aaubry/YamlDotNet
Module: https://github.com/cloudbase/powershell-yaml

**Comment by gitfvb** [comment]
Looks good so far. Currently in testing phase.

**Comment by gitfvb** [comment]
This is published now with 0.2.0
